pdns/docker-entrypoint.sh

```diff
@@ -12,17 +12,20 @@ fi
 : "${PDNS_gmysql_password:=${MYSQL_ENV_MYSQL_PASSWORD:-powerdns}}"
 : "${PDNS_gmysql_dbname:=${MYSQL_ENV_MYSQL_DATABASE:-powerdns}}"
 
-export PDNS_gmysql_host PDNS_gmysql_port PDNS_gmysql_user PDNS_gmysql_password PDNS_gmysql_dbname
+# use first part of node name as database name suffix
+if [ "${NODE_NAME:-}" ]; then
+    NODE_NAME=$(echo ${NODE_NAME} | sed -e 's/\..*//' -e 's/-//')
+fi
+PDNS_gmysql_dbname="${PDNS_gmysql_dbname}${NODE_NAME}"
```

Comment:
This won't work if `NODE_NAME` is not defined. I think this line should be moved to the condition above.